symfony/plugins/orangehrmCoreOAuthPlugin/lib/vendor/OAuth2/CHANGELOG.md

```diff
@@ -0,0 +1,200 @@
+CHANGELOG for 1.x
```

Comment:
@RajithaKumara any way to move this lib to composer.json and load via autoloader

Reply:
I'll try currently it's in last point of the task list

Reply:
nice .. just see the task list

Reply:
Done